folder-selection/src/App.tsx

```diff
@@ -0,0 +1,32 @@
+import FieldPlugin from './components/FieldPlugin'
+import { FunctionComponent } from 'react'
+import { FieldPluginProvider } from '@storyblok/field-plugin/react'
+
+import { CssBaseline, GlobalStyles, ThemeProvider } from '@mui/material'
+import { lightTheme } from '@storyblok/mui'
+
+const App: FunctionComponent = () => {
+  return (
+    <FieldPluginProvider
+      Loading={Loading}
+      Error={Error}
+    >
+      <ThemeProvider theme={lightTheme}>
+        <CssBaseline />
+        <GlobalStyles
+          styles={{
+            body: { backgroundColor: 'transparent' },
+          }}
+        />
+        <FieldPlugin />
+      </ThemeProvider>
+    </FieldPluginProvider>
+  )
+}
```

Comment:
```suggestion
const App: FunctionComponent = () => {
  const { type } = useFieldPlugin()
  
  if (type === 'loading') {
    return <Loading />
  } else if (type === 'error') {
    return <Error />
  } else {
    return (
      <ThemeProvider theme={lightTheme}>
        <CssBaseline />
        <GlobalStyles
          styles={{
            body: { backgroundColor: 'transparent' },
          }}
        />
        <FieldPlugin />
      </ThemeProvider>
    )
  }
}
```

Since v1.0.0 of Field Plugin SDK, we no longer have `FieldPluginProvider`. I think we can change this something like this:

Reply:
Done!